config/default.json

```diff
@@ -42,6 +46,10 @@
         "namePrefix": "",
         "nameSuffix": "",
         "minAreaSquareDeg": 1e-15
+      },
+      "validations": {
+        "namePrefix": "",
```

Comment:
why do we need prefix for validation table name, its may cause a problem with the db table name length limits

Reply:
we dont, but you commented above to add both suffix and prefix - i added them an the prefix is empty (to keep same template like in parts and polygonParts)